packages/components/molecules/f-mega-modal/README.md

```diff
@@ -181,17 +183,29 @@ Change directory to the `f-mega-modal` package:
 $ cd packages/components/molecules/f-mega-modal
 ```
 
-### Running storybook
+## Testing
+
+### Unit, Integration and Contract
 
-Storybook can be used to develop new and existing components.
+To test all components, run from root directory.
+To test only `f-mega-modal`, run from the `./fozzie-components/packages/f-mega-modal` directory.
```

Comment:
Did you want this to be a separate paragraph? If so, you'll need an extra line break for markdown to recognise it, otherwise it'll go up onto the previous line as part of the same paragraph.